### PR TITLE
Replace the description exporting to conversion

### PR DESCRIPTION
**Pull request by hwangdeyu**
fixes https://github.com/onnx/tensorflow-onnx/issues/1836, https://github.com/onnx/tensorflow-onnx/issues/1852
supplement of https://github.com/onnx/tensorflow-onnx/pull/1883
Minor changes to the document.

Signed-off-by: Deyu Huang <deyhuang@microsoft.com>
Co-authored-by: Emma Ning <>

### CODE REVIEW

**Review by EmmaNingMS — APPROVED** [review]

File: examples/tf_custom_op/custom_op.md [hunk 1]
```diff
@@ -99,11 +99,11 @@ After below steps, we can get a TensorFlow custom op library `double_and_add_one
...
 
-1- using [--load_op_libraries](https://github.com/onnx/tensorflow-onnx#--load_op_libraries) in conversion command or `tf.load_op_library()` method in code to load the TensorFlow custom ops library.
+1- use [--load_op_libraries](https://github.com/onnx/tensorflow-onnx#--load_op_libraries) in conversion command or `tf.load_op_library()` method in code to load the TensorFlow custom ops library.
 
-2- implement the op handler, registered it with the `@tf_op` decorator. Those handlers will be registered via the decorator on load of the module. [Here](https://github.com/onnx/tensorflow-onnx/tree/main/tf2onnx/onnx_opset) are examples of TensorFlow op hander implementations.
+2- implement the op handler according to the op definitions, registered it with the `@tf_op` decorator. Those handlers will be registered via the decorator on load of the module. [Here](https://github.com/onnx/tensorflow-onnx/tree/main/tf2onnx/onnx_opset) are examples of TensorFlow op hander implementations, which all are conbinded with ONNX ops.
```
Comment (fatcat-z) [inline examples/tf_custom_op/custom_op.md:106]: conbinded => combined.

Comment (hwangdeyu) [reply examples/tf_custom_op/custom_op.md]: Done.

File: examples/tf_custom_op/custom_op.md [hunk 2]
```diff
@@ -99,11 +99,11 @@ After below steps, we can get a TensorFlow custom op library `double_and_add_one
...
 
-1- using [--load_op_libraries](https://github.com/onnx/tensorflow-onnx#--load_op_libraries) in conversion command or `tf.load_op_library()` method in code to load the TensorFlow custom ops library.
+1- use [--load_op_libraries](https://github.com/onnx/tensorflow-onnx#--load_op_libraries) in conversion command or `tf.load_op_library()` method in code to load the TensorFlow custom ops library.
 
-2- implement the op handler, registered it with the `@tf_op` decorator. Those handlers will be registered via the decorator on load of the module. [Here](https://github.com/onnx/tensorflow-onnx/tree/main/tf2onnx/onnx_opset) are examples of TensorFlow op hander implementations.
+2- implement the op handler according to the op definitions, registered it with the `@tf_op` decorator. Those handlers will be registered via the decorator on load of the module. [Here](https://github.com/onnx/tensorflow-onnx/tree/main/tf2onnx/onnx_opset) are examples of TensorFlow op hander implementations, which all are conbined with ONNX ops.
```
Comment (fatcat-z) [inline examples/tf_custom_op/custom_op.md:106]: conbined -> combined?

**Review by fatcat-z — APPROVED** [review]
LGTM, thanks!

File: examples/tf_custom_op/custom_op.md (hunk 2 above)
Comment (hwangdeyu) [reply examples/tf_custom_op/custom_op.md]: Sure, thanks!

**Pull request merged**